crates/escalier/tests/errors/incompatible_types.error

```diff
@@ -41,13 +41,3 @@ Error: Incompatible types
    ·              ───┬───  
    ·                 ╰───── "world"
 ───╯
-
-Error: Incompatible types
-   ╭─[<unknown>:8:17]
-   │
- 8 │ let b: number = "5";
-   ·        ───┬──   ─┬─  
-   ·           ╰────────── number
-   ·                  │   
-   ·                  ╰─── "5"
-───╯
```

Comment:
This error will come back in a future PR when I add support for recovering from incorrect args being passed to a function call.